address.go

```diff
@@ -380,13 +380,17 @@ func (a *Address) MarshalCBOR(w io.Writer) error {
 func (a *Address) UnmarshalCBOR(r io.Reader) error {
 	br := cbg.GetPeeker(r)
 
-	pb, err := br.PeekByte()
+	pb, err := br.ReadByte()
 	if err != nil {
 		return err
 	}
 	if pb == cbg.CborNull[0] {
 		return nil
 	}
+	err = br.UnreadByte()
```

Comment:
I'm assuming that not un-reading (consuming the byte) in the `CborNull` case above has no side effect (and actually seems to me is what we should have been doing all along).

Reply:
PeekByte will unread internally. This change just gets rid of the abstraction to remove an indirect call.

Reply:
I.e., PeakByte is no more.